postgres/assets/configuration/spec.yaml

```diff
@@ -570,6 +570,24 @@ files:
           value:
             type: string
             example: us-east-1
+        - name: managed_authentication
+          description: |
+            Configure section used for AWS IAM Authentication with RDS.
+
+            This supports using IAM database authentication to connect to your database instance.
+            
+            For more information on RDS IAM Authentication, see the AWS docs
+            https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/UsingWithRDS.IAMDBAuth.Connecting.html
+
+            To enable IAM Authentication, set `aws.managed_authentication.enabled` to `true`.
+            If `aws.managed_authentication.enabled` is set, then the `username` and `password` fields will be ignored.
```

Comment:
`username` should not be ignored. The username is the same as the IAM role name, which doesn't have to be `datadog` necessarily

Reply:
Ohhh I blame copilot for this. I just hit tab

Reply:
This is why @alexandre-normand hates AI :shame-nun: